spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/GcpCloudSqlProperties.java

```diff
@@ -36,14 +36,6 @@
 
 	private String instanceConnectionName;
 
-	private String jdbcUrl;
-
-	private String jdbcDriver = "com.mysql.jdbc.Driver";
-
-	private String userName = "root";
-
-	private String password = "";
-
 	private boolean initFailFast;
```

Comment:
`initFailFast` can also go if we're not creating the Hikari datasource ourselves.